README.md

```diff
@@ -57,6 +57,13 @@ dnf copr enable erikreider/SwayNotificationCenter
 dnf install SwayNotificationCenter
 ```
 
+Fedora Silverblue (and other rpm-ostree variants):
+The package can be downloaded from COPR after adding the COPR repo as a ostree repo, and installed as a overlayed package:
+```zsh
+ostree remote add SwayNotificationCenter https://download.copr.fedorainfracloud.org/results/erikreider/SwayNotificationCenter/fedora-$releasever-$basearch/
```

Comment:
Are `$releasever` and `$basearch` already defined in the default fedora env or is this something that the user should fill in?

Also, if you're on version 36, how does updating to a newer version work (eg. 36 -> 37)? Will the repo URL be updated automatically?

Reply:
1. AFAIK, those variables point the package manager automatically to the version corresponding to your Fedora install. The user does not have to do anything.
2. The repo URL should always work to my understanding because of those variables. **However**, you won't be allowed to upgrade to a new version (like, 36 to 37) if any of your repos doesn't have a package for that new version. The upgrade will stop on its tracks and tell you this, and you'll be required to either remove the outdated repo or wait until that repo gets updated packages. You may also have to uninstall versioned (meant for a specific version) packages, or use packages that aren't versioned (not locked to a specific version). This last bit I just read it from [here](https://discussion.fedoraproject.org/t/simplifying-updates-for-rpm-fusion-packages-and-other-packages-shipping-their-own-rpm-repos) and haven´t actually tried, but I do know from experience that you won't be allowed to upgrade by default if matching packages aren't found.